### DISCUSSION TITLE
Right click within a tab to create split view

### DISCUSSION THREAD

**Discussion by thanhnguyen2187** [Floorp-Projects/Floorp#1224] [Feature Request]
Hi.

Thanks a lot for the awesome browser!

More context on why I need the functionality: I find it impossible to access "Fixed in Split View" from anywhere other than right clicking at the tab itself. It conflicts with my hiding of the native horizontal/vertical tab bar as I use Sidebery. I feel like adding the functionality to an item within context menu (right click) can be a good solution.

What do you think?

**Comment by breadbeforerice** [comment]
You can also create one from about:preferences#csk

**Reply by thanhnguyen2187** [reply]
Hi. There is no `#csk` on my `preferences`. Does it exist on newer versions (my current version is `11.13.2`)?

**Reply by creeper-0910** [reply]
It exists in the latest version.

**Reply by thanhnguyen2187** [reply]
Thanks! I'm using Floorp from NixOS, and it seems like the latest version (`14.x.x`) are not available. This should be resolved for now

**Reply by thanhnguyen2187** [reply]
Sorry for reopening this, but `#csk` doesn't work. When I start Floorp from the command line and try the split view key combination, it says:

```
JavaScript error: chrome://nora/content/assets/utils.js, line 5218: TypeError: window.gSplitView is undefined
```

Do you think there is any other thing I can do?
